packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -162,7 +163,10 @@ This app may not exist or you may not have permission to view it.''',
       logger.err(
         '''
 It looks like you have an existing ${platform.name} release for version ${lightCyan.wrap(release.version)}.
-Please bump your version number and try again.''',
+Please bump your version number and try again.
+
+You can manage this release in the Shorebird Console at:
```

Comment:
That way this just reads:

You can manage this release in the `[Shorebird Console](link-to-console)`